src/app/map-view/map-view.component.css

```diff
@@ -1,21 +1,79 @@
-*{
-  background-color: #7cc254;
-  margin: 0 0;
+.row {
+  margin-top: 1vh;
+  display: flex;
+}
+
+app-tracks {
+  flex: 30%;
+  margin-right: 0.5vw;
 }
 
-div.map{
-  float: right;
-  width: 80%;
-  margin-right: 5vw;
+.viewport{
+  background-color: #7cc254;
+  height: 100%;
+  flex: 70%;
+  margin: 0;
+  text-align: center;
+  font-size: 1.5vw;
 }
 
-#info{
-  padding: 0 0 40% 0.5%;
-  font-size: 0.9vw;
+.viewport.p {
+  padding: 0;
 }
 
-#pilot{
-  font-size: 1.1vw;
+.track-header{
+  font-size: 1.2vw;
   text-align: center;
-  padding: 1vh 0;
+  padding: 1vh 1vw;
+}
+
+.track-info-container {
+  display: flex;
+}
+
+.track-info {
+  flex: 33%;
+  font-size: 1vw;
+  text-align: left;
+}
+
+.tooltip-container {
+  visibility: hidden;
+  position: relative;
+  background-color: transparent;
+  width: 1px;
```

Comment:
I've tried using vw and vh to make the site responsive to size, do you think it's possible to convert?

Reply:
in this case it doesn't matter because it's pretty much always hidden, it's just an anchor for the tooltip.